core/src/main/java/org/gbif/pipelines/core/functions/transforms/DwCATermIdentifier.java

```diff
@@ -0,0 +1,149 @@
+package org.gbif.pipelines.core.functions.transforms;
+
+/**
+ * This enum helps in identifying the DwCATerms with its identifier
+ */
+public enum DwCATermIdentifier {
```

Comment:
the usage of this class can be replaced by org.gbif.dwc.terms.DwcTerm, it has class terms and GROUP_*

Reply:
i actually created it because i needed class identifiers(urls), which is not available in DwCTerm.